python_modules/libraries/dagster-airbyte/dagster_airbyte/resources.py

```diff
@@ -0,0 +1,222 @@
+import logging
+import time
+import json
+from typing import Optional
+
+import requests
+from dagster import (
+    get_dagster_logger, 
+    __version__,
+    resource,
+    StringSource,
+    Failure,
+    Field
+)
+from dagster_airbyte.types import AirbyteOutput
+from requests.exceptions import RequestException
+from enum import Enum
+
+DEFAULT_POLL_INTERVAL_SECONDS = 10
+
+class AirbyteState(str, Enum):
```

Comment:
hm I think this might actually be easier to use if it were not an actual enum (because it's kinda annoying to have to write `.value` everywhere). If this instead was just a bare class:

```
class AirbyteState:
    RUNNING = "running"
    SUCCEEDED = "succeeded"
    ...
```

then the functionality would stay the same and it'd be a little more ergonomic to use. Sorry for switching things up on you!

Reply:
Done. No worries @OwenKephart thanks!